<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountsWindow</class>
 <widget class="QDialog" name="AccountsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Konta</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>411</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="myAccountLabel">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight SemiConde</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Moje Konto</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="accNumberLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nr konta</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="accountsComboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="sumLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Saldo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sumLineEdit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>16</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="sendTransactionButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Wyślij przelew</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="goBackButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Wróć</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="myCreditButton">
      <property name="text">
       <string>Harmonogram kredytu</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="transactionLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Moje transakcje</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
